pkg/controllers/proxy/network_services_controller.go

```diff
@@ -240,6 +240,7 @@ type serviceInfo struct {
 	namespace                string
 	clusterIP                net.IP
 	port                     int
+	targetPort               string
```

Comment:
Why a string type? Shouldn't it have the same type as port?

Reply:
`TargetPort` in service spec is `intstr` type.